src/controller/AudioController.vala

```diff
@@ -0,0 +1,99 @@
+/*
+ * brickman -- Brick Manager for LEGO MINDSTORMS EV3/ev3dev
+ *
+ * Copyright 2014-2015 David Lechner <david@lechnology.com>
```

Comment:
Whoops, thought I got them all; looks like I completely forgot to change this header.